src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/AdjournmentAfterDateOrPeriod.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
```

Comment:
Worth moving these into an enum package. I'd probably move all the adjuournments into an adjournment one, then the enums in adjournment.enum

Reply:
done